src/util/interfaces.ts

```diff
@@ -0,0 +1,38 @@
+export interface AttachmentProps {
+  id: number;
+  file: string;
+  created: string;
+}
+export interface MessageProps {
+  id?: number;
+  text: string | null;
+  sender_username: string;
+  created: string;
+  attachments: Array<AttachmentProps>;
+  custom_json: object | string;
+}
+
+export interface PersonProps {
+  username: string;
+  first_name: string | null;
+  last_name: string | null;
+  avatar: string | null;
+  custom_json: string | object | null;
```

Comment:
what is custom_json

Reply:
Custom JSON is a Chat Engine feature https://www.youtube.com/watch?v=EcC6fA2Umn8&t=2s